### PR TITLE
Feature setup

### PR DESCRIPTION
**Pull request by Meggie-H**

### CODE REVIEW

File: README.md
Comment (yanevdw) [inline README.md:1]: Just be aware that your README is being overwritten by the README generated when launching a React, Vite and TS project.

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react-jsx",
+
+    /* Linting */
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noFallthroughCasesInSwitch": true
```
Comment (yanevdw) [inline tsconfig.json]: Here are some useful rules that ca be added:
```
"noImplicitAny": true,
"strictFunctionTypes": true,
"strictPropertyInitialization": true,
"noImplicitThis": true,
"alwaysStrict": true,
```

File: public/vite.svg
Comment (Hugovs2000) [inline public/vite.svg:1]: You can delete this, unless you want to use the svg somewhere 🤷🏻‍♂️

File: src/App.css
Comment (Hugovs2000) [inline src/App.css:1]: I believe this should be in your index.css file and then you no longer need the app.css file.

File: src/index.css
Comment (Hugovs2000) [inline src/index.css:1]: Put the imports that are in app.css in here.

**Review by Hugovs2000 — DISMISSED** [review]
All good for a setup. Just take a look at the app.css and index.css.

File: tsconfig.json (hunk 1 above)
Comment (Meggie-H) [reply tsconfig.json]: Please have a look at my changes

**Review by Hugovs2000 — APPROVED** [review]
Looks good now 👍🏻

**Review by yanevdw — APPROVED** [review]
The setup looks solid! Well done!

**Pull request merged**